rector.php

```diff
@@ -15,4 +15,10 @@
 	->withSkip([
 		__DIR__ . '/tests/stubs',
 	])
-	->withTypeCoverageLevel(0);
+	->withPreparedSets(
+		deadCode: true,
+		typeDeclarations: true,
```

Comment:
What is the benefit of this over withTypeCoverageLevel? Is it like maximum level?

Reply:
Yes